libs/auth/scopes/src/lib/api.scope.ts

```diff
@@ -13,4 +14,5 @@ export enum ApiScope {
   company = '@island.is/company',
   vehicles = '@island.is/vehicles',
   noDebtCertificate = '@island.is/no-debt-certificate',
+  ads = '@island.is/air-discount-scheme',
```

Comment:
Accidentally doubled scoped. Will be refactored

Reply:
I would refactor this to keep the `airDiscountScheme` and remove `ads`

Reply:
:+1: